wpilibj/build.gradle

```diff
@@ -111,6 +111,7 @@ dependencies {
     compile 'edu.wpi.first.ntcore:ntcore-java:+'
     compile 'org.opencv:opencv-java:3.2.0'
     compile 'edu.wpi.first.cscore:cscore-java:+'
+    compile 'systems.uom:systems-common-java8:0.7.1'
```

Comment:
It looks to me like some transitive dependencies are being used implicitly - it would be nice to call those out explicitly.